core/commands/commit.py

```diff
@@ -139,6 +140,97 @@ def run(self, edit):
             })
 
 
+class GsPedanticEnforceEventListener(EventListener):
+    """
+    Set regions to worn for Pedantic commits
+    """
+
+    def on_selection_modified(self, view):
+        if 'make_commit' not in view.settings().get('syntax'):
+            return
+
+        savvy_settings = sublime.load_settings("GitSavvy.sublime-settings")
+        if not savvy_settings.get('pedantic_commit'):
+            return
+
+        self.view = view
+        self.first_line_limit = savvy_settings.get('pedantic_commit_first_line_length')
+        self.body_line_limit = savvy_settings.get('pedantic_commit_message_line_length')
+        self.warrning_size = savvy_settings.get('pedantic_commit_warning_length')
```

Comment:
Typo in `warrning_size`; should this be `warning_size`, or even `warning_length` ?

Reply:
yes